src/Microsoft.Azure.WebJobs.Host/Listeners/FunctionListener.cs

```diff
@@ -140,11 +140,11 @@ public async Task StopAsync(CancellationToken cancellationToken)
                     {
                         string listenerName = _listener.GetType().FullName;
                         string functionName = _descriptor.LogName;
-                        _logger?.LogInformation($"Stopping the {listenerName} listener for function '{functionName}'");
+                        _logger?.LogInformation($"Stopping the listener:{listenerName} for function:{functionName}");
 
                         await _listener.StopAsync(cancellationToken);
 
-                        _logger?.LogInformation($"Stopped the {listenerName} listener for function '{functionName}'");
+                        _logger?.LogInformation($"Stopped the listener:{listenerName} for function:{functionName}");
```

Comment:
Two things with this log message:
1. Use structured logging instead of string interpolation. The SystemLogger will find `functionName` and use it in the correct column, for example.
2. Instead of a colon, wrap the listenerName and functionName in `' '` -- it makes it easier to see where the name begins and ends. 

so...
` _logger?.LogInformation("Stopped the listener '{listenerName}' for function '{functionName}'", listenerName, functionName);`